src/PdfReader/index.tsx

```diff
@@ -30,26 +33,48 @@ function pdfReducer(state: PdfState, action: PdfReaderAction): PdfState {
         ...state,
         isScrolling: action.isScrolling,
       };
+
+    case 'SET_RESOURCE':
+      return {
+        ...state,
+        resource: action.resource,
+      };
+
+    case 'SET_PAGENUM':
+      return {
+        ...state,
+        pageNumber: action.pageNum,
+      };
   }
 }
 
 export default function usePdfReader(args: ReaderArguments): ReaderReturn {
   const { webpubManifestUrl, manifest } = args ?? {};
-  const [state] = React.useReducer(pdfReducer, {
+
+  const [state, dispatch] = React.useReducer(pdfReducer, {
     type: 'PDF',
     colorMode: 'day',
     isScrolling: false,
     fontSize: 16,
     fontFamily: 'sans-serif',
+    resource: '',
```

Comment:
My personal preference would probably be to use `null` before any resource has been set as I think it more explicitly communicates what's happening and leaves less room for bugs, especially since `''` is seen as a valid string just like a resource url would be